.circleci/config.yml

```diff
@@ -159,6 +159,17 @@ jobs:
       - run: mvn versions:set -DnewVersion=${CIRCLE_TAG}
       - run: mvn deploy -Prun-revapi,public
 
+  dependency-track:
+    executor: java-image
+    steps:
+      - checkout
+
+      - restore_cache:
```

Comment:
Rather than restoring cache, we should attach to the workspace instead. This would be a better way if reducing repeated builds since we already restored cache in the build step.

```yaml
- attach_workspace:
          at: /root
```